### DISCUSSION TITLE
Is there a way to detect git bare repositories?

### DISCUSSION THREAD

**Discussion by jflores-p** [starship/starship#2331] [Q&A]
Hi, I have my dotfiles on a github repo from where I pull them to my pc using a bare repository, whith other prompts and shells I never found a way to track this "repo" which is basically mi .config/ folder and some other files in my home directory. But watching [this video](https://www.youtube.com/watch?v=LDLisRPAC_g), where I discovered this project, I saw that the guy also has his home directory as a git bare repo.
Reading his [dotfiles repo](https://gitlab.com/dwt1/dotfiles/-/tree/master) I didn't find any special config (in the general configuration of his setup) that could make the git_branch module work different than the standard way.

Do anyone know how he made it?

**Comment by pramirez-eztr** [comment]
```
[git_branch]  
ignore_bare_repo = true  # Don't show in bare repos
```
